### DISCUSSION TITLE
微信小程序能使用script标签的src属性引入js/ts吗?

### DISCUSSION THREAD

**Discussion by SilentFlute** [NervJS/taro#14714] [Q & A] [answered]
如题, 我用`taro 3.6.1`写了一个微信小程序, 有个`.vue`文件页面:
```
<template>
  //...
</template>

<script setup lang="ts">
//...
</script>

<style>
//...
</style>
```
然后页面代码比较多, 我就把样式文件拿了出去:
```
<template>
  
</template>

<script setup lang="ts">
//...
import './style.scss';
</script>
```
后来发现`ts`代码更多, 于是我打算将`ts`文件也拿出去, 看了下`vue`的文档: https://cn.vuejs.org/api/sfc-spec.html#src-imports, 之后我尝试通过`script`的`src`属性引入, 但一直报错说`export default`没找到, 但里面的代码是之前写在`<script setup>`里的代码, 由于是页面, 里面倒是没有`props` `emits`, 但有`export`的`ts`的`type`(也有没被`export`的类型), 还有各种变量和函数表达式

一时间不知道如何处理, 特来求教, 先行谢过

**Comment by yoyo837** [comment, marked as answer]
不能, 微信小程序并不支持从网络引入脚本

**Reply by SilentFlute** [reply]
哦哦, 好嘛, 我说怎么我用`<script src="./script.ts"></script>`试了但是不行, 还各种报错...
